2.4/examples/replica/mongodb-clustered.json

```diff
@@ -55,6 +55,23 @@
   ],
   "objects":[
     {
+      "kind": "PersistentVolumeClaim",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "mongodb-data-claim"
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteOnce"
+        ],
+        "resources": {
+          "requests": {
+            "storage": "2Gi"
```

Comment:
No. The problem is that PVCs are not "multiplied".  All pods get the same claim, backed by the same storage. Because of that, the current template in this PR does not work.

The value "2Gi" is not the most relevant bit here right now, and we can change it later. I just needed to push this to bring up the discussion on "how to scale PVCs".